exchanges/kucoin/kucoin_wrapper.go

```diff
@@ -670,8 +655,78 @@ func (ku *Kucoin) GetRecentTrades(ctx context.Context, p currency.Pair, assetTyp
 }
 
 // GetHistoricTrades returns historic trade data within the timeframe provided
-func (ku *Kucoin) GetHistoricTrades(_ context.Context, _ currency.Pair, _ asset.Item, _, _ time.Time) ([]trade.Data, error) {
-	return nil, common.ErrFunctionNotSupported
+func (ku *Kucoin) GetHistoricTrades(ctx context.Context, pair currency.Pair, assetType asset.Item, startTime, endTime time.Time) ([]trade.Data, error) {
```

Comment:
This actually still is not supported. You cannot get historic trades, which is why in your tests you don't have a start/end time in there. Please revert

Reply:
Removed! With apologies.